attributes/network/1_blockchain.adoc

```diff
@@ -31,3 +31,6 @@ In that case the payload of the response MUST be empty.
 
 === Block
 The `blockchain.block` endpoint returns a block and its metadata.
+
```

Comment:
You should also add it at the beginning of the document, in

```
It exposes the following endpoints (matching the Rosetta API):
...
```

Since you're there, you could also add the CDDL to mimic the other `adoc` files.